packages/design-system/src/components/NativeDialog/NativeDialog.tsx

```diff
@@ -26,7 +26,7 @@ interface NativeDialogProps extends Omit<DialogHTMLAttributes<HTMLElement>, 'chi
   showModal?: boolean;
 }
 
-const NativeDialog = ({
+export const NativeDialog = ({
```

Comment:
i didn't think we wanted to export this to users?

Reply:
If you take a look at the change in `packages/design-system/src/components/index.ts`, I'm removing the export from the package but adding it to this individual module just so I can import it in the doc site package.